### PR TITLE
Voting round

### PR DESCRIPTION
**Pull request by anoorbhai**

### CODE REVIEW

File: backups/database-backup.sql [hunk 1]
```diff
@@ -211,6 +211,40 @@ COPY public.word_pairs (id, civilian_word, undercover_word) FROM stdin;
...
+
+SELECT pg_catalog.setval('public.votes_id_seq', 1, false);
+
+--
+-- Name: votes votes_pkey; Type: CONSTRAINT; Schema: public; Owner: noahdisler
+--
+
+ALTER TABLE ONLY public.votes
+    ADD CONSTRAINT votes_pkey PRIMARY KEY (id);
+
```
Comment (Noah-Disler) [inline backups/database-backup.sql:247]: Nice addition - the votes table is well-structured and clearly captures the essential relationships for tracking round-based voting.

File: src/public/voting-round.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+        <!-- Section to display vote counts -->
+        <div class="card shadow-sm p-4 mt-4">
+            <h5 class="card-title">Vote Counts</h5>
+            <div id="vote-counts"></div>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (Noah-Disler) [inline src/public/voting-round.html:138]: This looks great - really clear voting flow and nice touch handling ties and result messages dynamically. I especially like how you integrated backend syncing while keeping the UI feedback responsive and smooth. Well done!

File: src/public/voting-round.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    voteCountsDiv.innerHTML = ""; 
+
+    for (const [username, voteCount] of Object.entries(votes)) {
+        const voteItem = document.createElement("p");
+        voteItem.textContent = `${username}: ${voteCount} votes`;
+        voteCountsDiv.appendChild(voteItem);
+    }
+}
+
+module.exports = { Player, VotingRound, updateVoteCounts };
```
Comment (Noah-Disler) [inline src/public/voting-round.js:84]: Really well-structured module - the class design is clean and readable, and the use of Map-like logic via Object.entries for votes is a great choice. I like how the elimination logic cleanly handles both ties and winners, and the updateVoteCounts function integrates nicely with the DOM.

File: src/routes/vote-routes.js
Comment (Noah-Disler) [inline src/routes/vote-routes.js:1]: Nice job

File: tests/vote-routes.test.js
Comment (Noah-Disler) [inline tests/vote-routes.test.js:1]: Solid coverage of both the success and failure cases!

File: tests/voting-round.test.js
Comment (Noah-Disler) [inline tests/voting-round.test.js:1]: Great coverage of all the core voting logic, including edge cases like invalid votes and ties. The assertions are clear and meaningful, and everything’s structured in a really readable way. Nice work!
